### PR TITLE
setup-environment: Allow for downstream EULA updates

### PR DESCRIPTION
**Pull request by thochstein**
Add a variable so NXP can update the click-through EULA when they
release their layers.

Signed-off-by: Tom Hochstein <tom.hochstein@nxp.com>

### CODE REVIEW

File: setup-environment [hunk 1]
```diff
@@ -131,6 +131,15 @@ if [ "$build_dir_setup_enabled" = "true" ] && [ -z "$DISTRO" ]; then
     return 1
 fi
 
+if [ "$FSL_EULA_FILE" = "" ]; then
+    FSL_EULA_FILE=$CWD/sources/meta-freescale/EULA
+fi
```
Comment (otavio) [inline setup-environment]:
```
[ -z "$FSL_EULA_FILE" ] && FSL_EULA_FILE=$CWD/sources/meta-freescale/EULA
```

**Review by otavio — CHANGES_REQUESTED** [review]

**Comment by thochstein** [comment]
Ping

**Comment by otavio** [comment]
Please extend the commit log explaining the reasoning and use case for this. I think I get the use case for it but it must be documented.

**Comment by thochstein** [comment]
> Please extend the commit log explaining the reasoning and use case for this. I think I get the use case for it but it must be documented.

Done.

**Review by otavio — APPROVED** [review]

**Pull request merged**